### PR TITLE
Added Volcano recommendations.

### PR DESCRIPTION
**Pull request by k82cn**
Signed-off-by: Klaus Ma <klaus1982.cn@gmail.com>

SIG-Runtime Sandbox request of Volcano (#3).

### CODE REVIEW

File: recommendations/sandbox/volcano.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+- v0.4 (2020 Q1): GPU sharing/topology, virtual kubelet
+- v1.0 (2020 Q2): Improve test coverage, Preemption/Reclaim Enhancement
+- v1.x (one release per quarter): Hierarchical Queue, Reservation, Backfill, Execution Time Estimation, Job dependency,
+  Plugins per Queue, Data Management, Data Aware Scheduling ......
+
+# Project Scope
+
+## Clear project definition
+
+Yes, the scope of Volcano is to build a Kuberentes native **batch system** to provide a suite of mechanisms that
```
Comment (dankohn) [inline recommendations/sandbox/volcano.md]: Kubernetes

File: recommendations/sandbox/volcano.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+## Value-add to the CNCF ecosystem
+
+More and more users would like to run batch workload (e.g. AI, BigData, HPC) in cloud native environment;
+Volcano is built as a Kubernetes based, cloud native batch system to help those batch workloads to
+use cloud native technologies. It's an enhanced orchestration layer to help batch workloads in CNCF ecosystem.
+
+## Alignment with other CNCF projects
+
+- Volcano helps kubernetes to run batch workloads natively as a kubernetes based batch system.
```
Comment (dankohn) [inline recommendations/sandbox/volcano.md]: Please capitalize Kubernetes everywhere or say K8s.

Comment (k82cn) [reply recommendations/sandbox/volcano.md]: sure :)

File: recommendations/sandbox/volcano.md (hunk 1 above)
Comment (k82cn) [reply recommendations/sandbox/volcano.md]: done :)

**Comment by k82cn** [comment]
comments addressed, thanks very much :)

**Comment by k82cn** [comment]
/cc @mrbobbytables 

As several users from [research user group](https://github.com/cncf/research-user-group) are also interesting in batch system on k8s :)

**Comment by amye** [comment]
Posted at https://youtu.be/krG8X6TYJ74

**Pull request merged**